common/src/main/java/org/apache/celeborn/common/network/protocol/PushMergedData.java

```diff
@@ -99,7 +99,7 @@ public static PushMergedData decode(ByteBuf buf, boolean decodeBody) {
 
   @Override
   public int hashCode() {
-    return Objects.hashCode(requestId, mode, shuffleKey, partitionUniqueIds, batchOffsets, body());
+    return Objects.hashCode(requestId, mode, shuffleKey);
```

Comment:
`requestId` is probably sufficient as a `hashCode` here.
Instead of hashing `partitionUniqueIds`, `batchOffsets` - I simply dropped them.
Same for `body`